src/modules/RootPart.rb

```diff
@@ -1711,7 +1712,20 @@ def MountPartitions(root_device_current)
 
               Builtins.y2milestone("fstab %1", fstab)
 
-              if !(
+              if reiserfs_in_fstab?(fstab)
+                reiserfs_entries = fstab.find_all {|e| e["vfstype"].include? "reiser" }
```

Comment:
1. Yep, rubocop is not used at all, but it's not a excuse to not respect the current conventions 👍 . Already modified.
2. 👍 
3. 👍